app/components/audit_log/row.js

```diff
@@ -204,6 +246,17 @@ export default Relay.createContainer(AuditLogRow, {
             ...on Pipeline {
               name
             }
+            ...on Team {
+              name
+            }
+            ...on TeamMember {
+              team { name }
+              user { name }
+            }
+            ...on TeamPipeline {
+              team { name }
+              pipeline { name }
```

Comment:
Considering I'm a stickler for hanging commas for the same reason, I endorse this product and/or service. :+1: